src/pages/Onboard/Resources.vue

```diff
@@ -225,7 +225,7 @@ export default {
       >
         <v-col cols="12" class="text-center py-2">
           <v-btn
-            :to="{ name: 'dashboard' }"
+            href="/"
```

Comment:
Just had a question, what is the difference between using :to="{path: '/'}" and href="/" ?

Reply:
Using the former handles the navigation through the router; the later is a full page refresh to the root directory. Doing this forces a refresh of the application (it's equivalent to retyping cloud.prefect.io into the url bar) and ensures local state is cleared after the license has been created.